dispatch/once.h

```diff
@@ -38,7 +38,7 @@ __BEGIN_DECLS
  * Note: static and global variables default to zero.
  */
 DISPATCH_SWIFT3_UNAVAILABLE("Use lazily initialized globals instead")
-typedef long dispatch_once_t;
+typedef intptr_t dispatch_once_t;
```

Comment:
should really be `uintptr_t` I don't like signed integers for these.

Reply:
oh right and since it's API it may cause build fallouts... ok convinced let's keep intptr_t ugh